src/mmw/sass/pages/_data-catalog.scss

```diff
@@ -46,9 +69,9 @@
         }
         .data-catalog-search-icon {
             color: $ui-grey;
-            position: relative;
```

Comment:
We have a card to fix pagination #2091. The date filter opens and closes correctly, pushing things down and up as it should. Don't think this is an issue here.